documentation/developer/language/02_variables.md

```diff
@@ -35,3 +36,33 @@ const a: u32 = 0;
 let b: u32 = 0;
 b = 1; // <- Ok
 ```
+
+## Global Constants
+
+Leo allows defining global constants in the main scope of a program. Syntax for these definitions is the same as for
+constants inside the function:
+
+```leo
+const MAGIC_NUMBER: u8 = 42;
+
+function main() -> u8 {
+    // here you can access these variables
+    return MAGIC_NUMBER;
+}
+
+function some_other_func() {
+    // constants are accessible everywhere
+    let num = MAGIC_NUMBER;
+}
+```
+
+:::info
+Global constants only exist in the scope of the file where they are defined. Import of these constants is not allowed.
+:::
+
+Constants can also be used in the other constant definitions:
```

Comment:
Can you specify here that using constants in other constants is top-down only? I.e., can define a const that uses a const after that one is defined.

Reply:
Yes, important point.